app/views/assignments/_cp_assignment_upload_modal.html.erb

```diff
@@ -8,21 +8,21 @@
 
   <!-- Script for disabling upload buttons. -->
   <%= javascript_include_tag 'upload_button_control.js',
-                             id: 'upload_script_for_config',
-                             upload_id: 'upload_config_files',
+                             id: 'upload_script',
+                             upload_id: 'upload_files_for_config',
                              button_id: 'submit_upload_for_config',
                              nonce: true %>
 
-  <%= form_tag upload_config_files_assignment_path(:id),
-               { multipart: true, size: 1 } do %>
```

Comment:
why remove the size?

Reply:
I did not know what it did (I had thought it limited the upload capacity to one file but that was wrong) and when I passed the equivalent html into an html validator (as I was debugging the disabled buttons issue), it raised an error so I removed it to avoid any unnecessary code

Reply:
Got it. If you want to limit it to one file, then don't set the `:multiple` argument on the `file_field_tag` (https://apidock.com/rails/ActionView/Helpers/FormTagHelper/file_field_tag)